package.json

```diff
@@ -22,6 +22,7 @@
     "build:libs:force": "pnpm -r --filter=\"!./apps/**\" --filter=\"!ledger-live\" build",
     "build:actions": "pnpm turbo run build --filter=@actions/* --no-deps",
     "build:cli": "pnpm turbo run build --filter=live-cli",
+    "build:llc": "pnpm turbo run build --filter=\"!./apps/**\" --scope=@ledgerhq/live-common --include-dependencies --no-deps",
```

Comment:
Could be useful to have a `build:llc:deps` as well, current alternative is `build:libs` but it also builds _libs/ui_ I think...

Reply:
at the moment i don't have the need here. i'm not exactly sure what would be the need of this neither 🤔